webpack.common.js

```diff
@@ -0,0 +1,29 @@
+const path = require('path');
+/* eslint-disable */
```

Comment:
- [ ] The previous reviewer asked us to reenable linters and address the issue pointed out by the linter. This appears not to have been addressed, please look into it one more time.